pkg/controller/operators/operatorcondition_controller.go

```diff
@@ -201,6 +202,13 @@ func (r *OperatorConditionReconciler) ensureDeploymentEnvVars(operatorCondition
 		if err != nil {
 			return err
 		}
+
+		// Check the deployment is owned by a CSV with the same name as the OperatorCondition.
+		deploymentOwner := ownerutil.GetOwnerByKind(deployment, operatorsv1alpha1.ClusterServiceVersionKind)
```

Comment:
Would it be easier to use `IsOwnedBy` instead of this whole if condition?

Reply:
I would rather not have to make a get call against the api server to check this.

Reply:
Fair enough.